Description: Don't have a "file manager" entry in the Open with dialogues
Bug: https://bugzilla.xfce.org/show_bug.cgi?id=7257
Bug-Ubuntu: https://bugs.launchpad.net/ubuntu/+source/exo/+bug/1310979

Index: exo-0.10.3/exo-open/exo-file-manager.desktop.in
===================================================================
--- exo-0.10.3.orig/exo-open/exo-file-manager.desktop.in	2012-12-27 13:50:31.000000000 +0100
+++ exo-0.10.3/exo-open/exo-file-manager.desktop.in	2014-02-28 15:00 +0200
@@ -7,6 +7,6 @@
 Terminal=false
 Categories=Utility;X-XFCE;X-Xfce-Toplevel;
 OnlyShowIn=XFCE;
-X-XFCE-MimeType=x-scheme-handler/file;x-scheme-handler/trash;
+X-XFCE-MimeType=inode/directory;x-scheme-handler/trash;
 _Name=File Manager
 _Comment=Browse the file system
